packages/admin-panel-server/src/routes/FetchEntityHierarchyTreeRoute.ts

```diff
@@ -44,19 +38,16 @@ export class FetchEntityHierarchyTreeRoute extends Route<FetchEntityHierarchyTre
 
     const childRelations = await centralApi.fetchResources('entityRelations', {
       filter: { parent_id: rootEntities.map(({ id }) => id) },
-      columns: ['parent_id', 'entity.code', 'entity.name'],
-      sort: ['entity.name'],
+      columns: ['parent_id', 'child_id'],
     });
     const childRelationsByParent = groupBy(childRelations, 'parent_id');
 
     return rootEntities.map(({ id, code, name }) => ({
+      id,
       code,
       name,
-      hierarchyName: code,
```

Comment:
`hierarchyName` is not required eventually, neither are all other children props except for `id`